CMakeLists.txt

```diff
@@ -39,6 +39,9 @@ FetchContent_MakeAvailable(stb)
 FetchContent_Declare(assimp GIT_REPOSITORY https://github.com/assimp/assimp.git)
 FetchContent_MakeAvailable(assimp)
 
+cmake_path(SET DATA_DIR "${PROJECT_SOURCE_DIR}")
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/src/data_dir.h.in" "${CMAKE_CURRENT_BINARY_DIR}/src/data_dir.h")
```

Comment:
This is 🏅 

Thank you. I completely forgot that all paths are absolute to my machine.